module/behaviour/strategy/SoccerStrategy/data/config/SoccerStrategy.yaml

```diff
@@ -1,10 +1,10 @@
 log_level: WARN
 
-ball_last_seen_max_time: 10
+ball_last_seen_max_time: 2
```

Comment:
I'm assuming that these are seconds, adding some comments for the units would be useful for context in my opinion (particularly if implementation changes down the line and this value changes to represent something less obvious)